custom/logistics/management/commands/clear_verified_numbers.py

```diff
@@ -0,0 +1,13 @@
+from django.core.management import BaseCommand
+from corehq.apps.sms.mixin import VerifiedNumber
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        logistics_domains = ['ewsghana-test-1', 'ilsgateway-test-1', 'ilsgateway-test-2']
+        for domain in logistics_domains:
```

Comment:
@kkrampa I was thinking more along the lines of:

    for domain in args:
        ...

that way we can use it again for any domains in the future too (and we can also move it to corehq/apps/sms/management/commands). I just wanted to make sure we had a way to explicitly list the domains so we don't have any mix-up.